server/package.json

```diff
@@ -55,8 +55,10 @@
     "@types/mocha": "^10.0.1",
     "@types/supertest": "^2.0.12",
     "chai": "^4.3.7",
+    "concurrently": "^7.6.0",
     "cross-env": "^7.0.3",
     "mocha": "^10.2.0",
+    "nyc": "^15.1.0",
```

Comment:
I don't see these new packages (`concurrently`, `nyc`) being used, are they necessary/relevant?

Reply:
must have leaked from another branch lol

Reply:
also, yeah the workflow is needed because okteto requires different configurations for preview and staging. preview is all automatic but for some reason, staging is not